docs/writing/posts/learning.md

```diff
@@ -0,0 +1,153 @@
+
+#### Learn from people actively doing the thing 'in competition'
+
+You should always try to learn from people who are actively doing the thing that you want to do. There's some nuance too around learning from a range of people and skills too. A group of ten white belts (newest) is probably going to be a terrible and difficult path to learning, but having a mix of black belts might also be technically difficult to learn from.
+
+If you want to succeed in business, you should learn from people who are actively engaged in business. This might not be an investor, and it might not be Zuckerberg either. You need to find the right blend of individuals who are at your level, a few months ahead, and significantly more advanced, actually pursuing the things you want to do in the current moment. There are many teachers who are out of touch or too far removed from the moment they were learning in.
+
+### Rocket League
+
+Rocket League is a video game where you play soccer with cars that can fly. The ability to fly is a critical aspect as it allows for a greater range of movement in the air, effectively enabling gameplay in six dimensions. It's difficult to explain, but I can share a [link](https://www.youtube.com/watch?v=pC_LFdmWHAU) of a professional player demonstrating the game.
+
+It took me two years of BJJ to appreciate the importance of strong fundamentals. When I picked up Rocket League during the pandemic, I decided to focus solely on the basics. Initially, I thought this would involve mastering the aerial flying aspect of the game. However, after watching numerous tutorials on YouTube, I discovered that the key was to drive fast and consistently hit the ball. Duh! All the tutorials unanimously advised against learning how to get airborne, as it often led to unforced errors. By the fourth week of playing, dedicating around 30 to 40 minutes a day, I had already reached the Diamond rank. I never tried to leave the air, and usually when I did, I would get scored on...
+
+#### Focus on the Fundamentals
+
+Half the time all of life is just avoiding unforced errors. I've realized that much of my success, my entire life, comes down to doing the simple and obvious things for a long time. I've witnessed, game after game, low-ranked players attempted fancy moves and failed, allowing me to capitalize on their mistakes.
+
+The road to success can be incredibly mundane. Interestingly, Rocket League was the first time I truly embraced this notion from the start. I told myself that I wanted to win, and then I started winning. I discovered that I could achieve a reasonably high level of success by simply doing two things well: steering effectively and making good contact with the ball. It's crucial not to overextend and commit unforced errors. I am convinced that most people fail to reach their desired goals due to unforced errors alone.
+
+Here are a few examples of silly unforched errors:
```

Comment:
I feel like you should just divide this section into "Focus on the Fundamentals" and then "Avoid unforced errors".

Reply:
i really do think they are actually the same, avoiding unforced errors is the fundamentals.